<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProjectExplorer::Internal::ProjectExplorerSettingsPageUi</class>
 <widget class="QWidget" name="ProjectExplorer::Internal::ProjectExplorerSettingsPageUi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>437</width>
    <height>389</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QGroupBox" name="directoryGroupBox">
     <property name="title">
      <string>Projects Directory</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::ExpandingFieldsGrow</enum>
      </property>
      <item row="1" column="1">
       <widget class="Utils::PathChooser" name="projectsDirectoryPathChooser" native="true"/>
      </item>
      <item row="0" column="0" colspan="2">
       <widget class="QRadioButton" name="currentDirectoryRadioButton">
        <property name="text">
         <string>Current directory</string>
        </property>
        <attribute name="buttonGroup">
         <string>directoryButtonGroup</string>
        </attribute>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QRadioButton" name="directoryRadioButton">
        <property name="text">
         <string>Directory</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
        <attribute name="buttonGroup">
         <string>directoryButtonGroup</string>
        </attribute>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="buildAndRunGroupBox">
     <property name="title">
      <string>Build and Run</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QCheckBox" name="saveAllFilesCheckBox">
        <property name="text">
         <string>Save all files before build</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="buildProjectBeforeDeployCheckBox">
        <property name="text">
         <string>Always build project before deploying it</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="deployProjectBeforeRunCheckBox">
        <property name="text">
         <string>Always deploy project before running it</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="showCompileOutputCheckBox">
        <property name="text">
         <string>Show compiler output on building</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="showRunOutputCheckBox">
        <property name="text">
         <string>Show run output on running</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="cleanOldAppOutputCheckBox">
        <property name="text">
         <string>Clear old application output on a new run</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="wrapAppOutputCheckBox">
        <property name="text">
         <string>Word-wrap application output</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="QCheckBox" name="jomCheckbox">
          <property name="whatsThis">
           <string/>
          </property>
          <property name="text">
           <string>Use jom instead of nmake</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="jomLabel">
          <property name="text">
           <string>&lt;i&gt;jom&lt;/i&gt; is a drop-in replacement for &lt;i&gt;nmake&lt;/i&gt; which distributes the compilation process to multiple CPU cores. The latest binary is available at &lt;a href=&quot;ftp://ftp.qt.nokia.com/jom/&quot;&gt;ftp://ftp.qt.nokia.com/jom/&lt;/a&gt;. Disable it if you experience problems with your builds.</string>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
          <property name="indent">
           <number>20</number>
          </property>
          <property name="openExternalLinks">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Utils::PathChooser</class>
   <extends>QWidget</extends>
   <header location="global">utils/pathchooser.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="directoryButtonGroup"/>
 </buttongroups>
</ui>
